frontend/src/app/offers/create-offer/create-offer.component.html

```diff
@@ -0,0 +1,299 @@
+          </div>
+        </div>
+      </div>
+      </div>
+      <div class="col-sm-3">Określ ramy czasowe (w formacie RRRR-MM-DD) oraz liczbę wolontariuszy, odpowiednio dla rezerwacji podstawowej i rozerwowej.</div>
+  </div>
+  <div class="form-group row">
+    <label for="description" class="col-sm-3 col-form-label">Opis oferty:</label>
+    <div class="col-sm-6">
+      <textarea
+        type="text"
+        class="form-control"
+        placeholder="Opis oferty..." 
+        minlength="3"
+        maxlength="500"
+        id="description" name="description"
+        [(ngModel)]="offer.description"
+        #description="ngModel"
+        required>
+      </textarea>
+      <span class="form-text text-muted" *ngIf="!description.valid &&description.touched">To pole jest obowiązkowe.</span>
+    </div>
+    <div class="col-sm-3">Postaraj się dokładnie opisać akcję.</div>
+  </div>
+  <div class="form-group row">
+      <label for="timeCommitment" class="col-sm-3 col-form-label">Wymagania czasowe:</label>
+      <div class="col-sm-6">
+        <input 
+          type="text"
+          class="form-control"
+          placeholder="Wymagania czasowe..." 
+          minlength="3"
+          maxlength="150"
+          id="timeCommitment" name="timeCommitment"
+          [(ngModel)]="offer.timeCommitment"
+          #timeCommitment="ngModel"
+          required>
+        <span class="form-text text-muted" *ngIf="!timeCommitment.valid &&timeCommitment.touched">To pole jest obowiązkowe.</span>
+      </div>
+      <div class="col-sm-3">Określ wymagania czasowe - ile godzin pracy w ciągu dnia wymaga dana akcja.</div>
+    </div>
+  <div class="form-group row">
+    <label for="benefits" class="col-sm-3 col-form-label">Korzyści:</label>
+    <div class="col-sm-6">
+      <textarea
+        type="text"
+        class="form-control"
+        placeholder="Korzyści dla wolontariusza..." 
+        minlength="3"
+        maxlength="500"
+        id="benefits" name="benefits"
+        [(ngModel)]="offer.benefits"
+        #benefits="ngModel"
+        required>
+      </textarea>
+      <span class="form-text text-muted" *ngIf="!benefits.valid &&benefits.touched">To pole jest obowiązkowe.</span>
+    </div>
+    <div class="col-sm-3">Określ korzyści płynące z udziału w akcji.</div>
+  </div>
+  <div class="form-group row">
+    <label for="requirements" class="col-sm-3 col-form-label">Wymagania:</label>
+    <div class="col-sm-6">
+      <input 
+        type="text"
+        class="form-control"
+        placeholder="Wymagania wobec wolontariusza..." 
+        minlength="3"
+        maxlength="150"
+        id="requirements" name="requirements"
+        [(ngModel)]="offer.requirements"
+        #requirements="ngModel"
+        required>
+    </div>
+    <div class="col-sm-3">Określ warunki, jakie musi spełnić kandydat, aby wziąć udział w akcji.</div>
+  </div>
+  <div class="d-flex justify-content-center">
+    <div *ngIf="!edit; else saveChanges">
```

Comment:
No need for ngIf and another template here. Leave just one button tag and write inside the button:
`Zapisz {{ inEditMode ? "zmiany" : "oferte" }}`

Also, since we removed native browser validation on form, we need to disable the button if the form is invalid. Therefore we need to add attribute binding to the button:
`[disabled]="!offerForm.form.valid"`